src/content/reference/hi/p5/erase.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: erase
+module: Color
```

Comment:
Here also translate module and submodule

Reply:
Done